### DISCUSSION TITLE
How to add "produces" to the endpoint OpenAPI documentation?

### DISCUSSION THREAD

**Discussion by vanyakosmos** [fastapi/fastapi#4760] [Questions] [answered]
`produces` describes types that endpoint can accept. [Extending](https://fastapi.tiangolo.com/advanced/extending-openapi/) whole openapi schema seams like overkill.

**Comment by vanyakosmos** [comment, marked as answer]
never mind, if someone have enough permissions - delete this please

**Reply by scrooks** [reply]
Did you figure out how to do this?  I want to add `produces` so that the docs will let me send in something other than `application/json` for an endpoint.
